website/docs/Installation.md

```diff
@@ -1,5 +1,14 @@
 # Installation
 
+## Setup Virtual Environment
```

Comment:
Thank you @Ayushpanditmoto , could you please also add instructions on how to set up env with conda?